test/mod/test_api.py

```diff
@@ -79,6 +79,28 @@ def test_get_arguments(self):
             self.assertEqual(data, args)
 
 
+    def test_exception(self):
+        # Check that 'api.exception' correctly sets 'API.exception'
+        tmpdir = self.tmp.name
+        path = os.path.join(tmpdir, "osbuild-api")
+        args = {}
+        monitor = osbuild.monitor.BaseMonitor(sys.stderr.fileno())
+
+        def exception(path):
+            osbuild.api.exception(path)
+            return 0
```

Comment:
`api.exception` does `sys.exit`, so I guess instead of `return` we should make sure that this line is never reached.